composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/ui/components/BalanceCard.kt

```diff
@@ -73,6 +73,15 @@ private fun BalanceContent(amount: Int) {
         ),
         color = MaterialTheme.colors.onSurface
     )
+    if (amount < 20) {
+        Spacer(modifier = Modifier.height(8.dp))
+        Text(
+            text = "Your Card Balance is lower than ৳20, Remember to recharge for a smooth journey",
```

Comment:
`Your Card Balance is lower than ৳20,` seems a bit redundant because the balance is displayed right above. I suggest we turn the color of the balance red instead.

Text here can simply read around the lines of:

- "Low balance: Below minimum fare. Please recharge."
- "Balance too low for next trip. Top up needed."
- "Insufficient funds. Please add fare."
- "Recharge required - Balance below minimum fare."
- "Can't ride with current balance. Please reload."

Reply:
Yup, Both sound good. will push a quick fix.

Reply:
Pushed.